src/AltinnCore/UnitTest/Runtime/data/DataAndSubmitTasks.bpmn

```diff
@@ -6,18 +6,17 @@
   xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
   xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
   xmlns:altinn="http://altinn"
-  xmlns:tjeneste="http://tjeneste"
   xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd"
   targetNamespace="http://bpmn.io/schema/bpmn" >
   <bpmn2:process id="Process_1" isExecutable="false">
     <bpmn2:startEvent id="StartEvent_1">
       <bpmn2:outgoing>SequenceFlow_1</bpmn2:outgoing>
     </bpmn2:startEvent>
-    <bpmn2:task id="FormFilling_1" name="FormFilling" altinn:steptype="data" altinn:egenskap1="egenskap1" altinn:egenskap2="egenskap2">
+    <bpmn2:task id="FormFilling_1" name="Data" altinn:tasktype="data">
```

Comment:
this might not break in the test cases, but changing the task name from FormFilling to data for apps will break runtime.

Reply:
React app is changed and should use new process api which handles this.